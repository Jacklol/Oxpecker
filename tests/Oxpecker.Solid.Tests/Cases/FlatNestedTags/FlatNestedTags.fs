module Oxpecker.Solid.Tests.Cases.FlatNestedTags

open Oxpecker.Solid

[<SolidComponent>]
let FlatNestedTags () =
    body() {
        "zero"
        h1(id="one")
        div(id="two") { "2" }
        div() { "3" }
        h1()
        "five"
    }
