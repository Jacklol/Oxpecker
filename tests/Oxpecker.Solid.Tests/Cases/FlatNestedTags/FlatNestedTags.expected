
export function FlatNestedTags() {
    return <body>
        <h1 id="one"></h1>
        <div id="two">
            2
        </div>
        <div>
            3
        </div>
        <h1></h1>
    </body>;
}

